### PR TITLE
Formatting py (do not merge for now)

### PR DESCRIPTION
**Pull request by hyp3rion31gr3y**
Solves: [https://issues.redhat.com/browse/INSIGHTSQA-3910](https://issues.redhat.com/browse/INSIGHTSQA-3910)

cc: @jhutar

### CODE REVIEW

**Comment by jhutar** [comment]
Could you also add MYPY config file https://mypy.readthedocs.io/en/stable/type_inference_and_annotations.html#other-ways-to-silence-errors to silence these complaints about variable type? That seems to be too big effort for us.

File: .github/linters/.flake8 [hunk 1]
```diff
@@ -0,0 +1 @@
+extend-ignore = ABC123
```
Comment (jhutar) [inline .github/linters/.flake8:1]: Is `ABC123` some actual flake8 error code?

Comment (hyp3rion31gr3y) [reply .github/linters/.flake8:1]: Actually forgot to remove those lines of code. We were referring to [https://stackoverflow.com/questions/73568036/flake8-line-break-before-binary-operator-how-to-fix-it](https://stackoverflow.com/questions/73568036/flake8-line-break-before-binary-operator-how-to-fix-it)

But fixed here [https://github.com/redhat-performance/opl/pull/65/files](https://github.com/redhat-performance/opl/pull/65)

**Comment by jhutar** [comment]
Closing. Discussed with Raj on Slack.

**Pull request closed without merging**